pygmt/src/basemap.py

```diff
@@ -61,6 +62,24 @@ def basemap(self, **kwargs):
         [**g**\|\ **j**\|\ **J**\|\ **n**\|\ **x**]\ *refpoint*\
         **+w**\ *length*.
         Draws a simple map scale centered on the reference point specified.
+    box : bool or str
+        [**+c**\ *clearances*][**+g**\ *fill*][**+i**\ [[*gap*/]\ *pen*]]\
+        [**+p**\ [*pen*]][**+r**\ [*radius*]][**+s**\ [[*dx*/*dy*/][*shade*]]].
+        If set to ``True``, draws a rectangular border around the map inset,
+        map scale or rose. Alternatively, specify a different pen with
```

Comment:
`map inset` is no longer available in the modern-mode "basemap" module. Users who wants to use `inset` should use the `inset` module instead.
```suggestion
        If set to ``True``, draws a rectangular border around the 
        map scale or rose. Alternatively, specify a different pen with
```